### DISCUSSION TITLE
Modification of genesis python source code does not take effect ~

### DISCUSSION THREAD

**Discussion by hyp1998** [Genesis-Embodied-AI/genesis-world#398] [Q&A] [answered]
I modified the genesis source code, but it did not take effect when I ran it. I want to know, do I need to compile after modifying the source code?Or is there something else that will take effect and I'm changing the default value of far in camera.py from 100 to 50

**Comment by hyp1998** [comment]
> 我修改了genesis源代码，但是运行的时候没有生效，我想知道修改源代码后需要编译吗？还是有其他什么会生效，我正在把camera.py中far的默认值从100改成50

@zhouxian

**Comment by zhouxian** [comment]
If you installed via ‘pip install -e .’ then you don’t need to do anything - the change will take effect

**Reply by hyp1998** [reply]
I downloaded the genesis source code via git

**Reply by hyp1998** [reply]
pip install genesis-world
This instruction was also executed

**Reply by zhouxian** [reply, marked as answer]
oh no, if you download the source code, you should do pip install -e . Inside genesis folder
